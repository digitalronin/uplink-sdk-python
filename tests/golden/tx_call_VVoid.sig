r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 101160095917267117608919311709895274744272229584632299579392202483009246963812
