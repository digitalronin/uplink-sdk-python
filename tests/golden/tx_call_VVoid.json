{
    "header": {
        "tag": "TxContract",
        "contents": {
            "tag": "Call",
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "method": "get",
                "args": [
                    {
                        "tag": "VVoid"
                    }
                ]
            }
        }
    },
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0yNDE0ODU3OTQyNDM4MjEzOTI4ODQwOTAzMzUyOTU4NTIzMzEwODAxMzQxNTA5NjY5MjUzNTY0MjA4OTg2Mjk0MjU2OTE2MDQxOTQ1NA==",
    "timestamp": 1231006505,
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "to": null
}